db/migrate/20220120201907_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
```

Comment:
- [ ] Optional suggestion

- I would recommend renaming/changing the columns like `comments_counter` and `likes_counter`.

Reply:
Thank you for the suggestion 👍